docs/sources/developers/_index.md

```diff
@@ -6,4 +6,10 @@ weight = 190
 
 # Developers
 
-This section of the documentation contains pages with resources for Grafana developers.
+This section of the documentation contains pages with resources for Grafana developers:
+
+- [Grafana Contribution Guide - learn how you can contribute your skills to make Grafana even better.](contribute)
+- [How to build a plugin](plugins)
```

Comment:
Please use relref format for internal links, https://github.com/grafana/grafana/tree/main/docs#using-relref-for-internal-links

Reply:
@marefr done